docs/devel/faster_reviews.md

```diff
@@ -53,15 +53,24 @@ later, just as soon as they have more free time (ha!).
 
 Let's talk about how to avoid this.
 
+## 0. Familiarize yourself with project conventions
+
+* [Development guide](development.md)
+* [Coding conventions](coding-conventions.md)
+* [API conventions](api-conventions.md)
+* [Kubectl conventions](kubectl-conventions.md)
+
 ## 1. Don't build a cathedral in one PR
 
 Are you sure FeatureX is something the Kubernetes team wants or will accept, or
 that it is implemented to fit with other changes in flight?  Are you willing to
 bet a few days or weeks of work on it?  If you have any doubt at all about the
-usefulness of your feature or the design - make a proposal doc or a sketch PR
-or both.  Write or code up just enough to express the idea and the design and
-why you made those choices, then get feedback on this.  Now, when we ask you to
-change a bunch of facets of the design, you don't have to re-write it all.
+usefulness of your feature or the design - make a proposal doc (in docs/proposals)
+or a sketch PR (e.g., just the API or Go interface) or both.  Write or code up
+just enough to express the idea and the design and why you made those choices,
+then get feedback on this. Be clear about what type of feedback you are asking
+for. Now, when we ask you to change a bunch of facets of the design, you don't
```

Comment:
How about "if we ask you to change..."
to make us sound nicer, even if the current wording is more accurate.